src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -525,45 +529,59 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     } getOrElse Future.successful(())
   }
 
-
   private def validateClusterRequestBucketObjectUri(userEmail: WorkbenchEmail, googleProject: GoogleProject, clusterRequest: ClusterRequest)(implicit executionContext: ExecutionContext): Future[Unit] = {
-    for {
+    val transformed = for {
+      // Get a pet token from Sam. If we can't get a token, we won't do validation but won't fail cluster creation.
```

Comment:
Yeah, it's debatable how valuable these checks really are. They are here to fail fast -- e.g. if a user provides a bogus bucket URI it'll fail at API request time whereas otherwise it'll fail in the init script ~5 mins later. The validations are not critical though -- so my thinking was to make transient errors in validation _not_ fail the cluster request.